qiskit/providers/models/pulsedefaults.py

```diff
@@ -118,7 +120,7 @@ def __init__(self,
                  qubit_freq_est: List[float],
                  meas_freq_est: List[float],
                  buffer: int,
-                 pulse_library: List[PulseLibraryItem],
```

Comment:
Why?

Reply:
I think because I was hitting a failure when I left it in. But that was probably because the import path changed, I'll look at adding it back now.

Reply:
Ok, I added this back in: https://github.com/Qiskit/qiskit-terra/pull/3383/commits/e084891d457e5ba16f8fb8a1ea58da9c32395b33 sorry it took so long